pxblue/angular-component-library
# Add tooltips to collapsed persistent drawer

Fixes #301 .


#### Changes proposed in this Pull Request:
- Add tooltip to the closed drawer persistent variant
- Add disableRailTooltip know to full config storybook story


#### Screenshots:


#### To Test:
- Run storybook
- Go to Drawer full config story
- Fiddle around with the `disableRailTooltip` and `openOnHover` props to see various states of tooltips
